lib/browser_client.dart

```diff
@@ -41,16 +41,17 @@ class BrowserClient extends BaseClient {
   bool withCredentials = false;
 
   /// Sends an HTTP request and asynchronously returns the response.
-  Future<StreamedResponse> send(BaseRequest request) async {
-    var bytes = await request.finalize().toBytes();
+  Future<Response> send(Request request) async {
+    var stream = new ByteStream(request.read());
```

Comment:
I really want to kill `ByteStream`, so we shouldn't use it here. You can pull this functionality into a function in `utils/`, although really it should live in a package somewhere.